packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -731,15 +747,38 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
 
       // timeouts
       sendTimeoutAfterOvalDrawingDelay: actions.send(
-        { type: 'TIMEOUT' },
+        {
+          type: 'RUNTIME_ERROR',
+          data: {
+            message: 'Client failed to draw oval.',
```

Comment:
Do we have a constants file where we store these error message? Could we move them out to a const as a follow up item?